packages/jet-brains-integration/src/web-types-generator.ts

```diff
@@ -55,49 +55,56 @@ export function getTagList(
           description: slot.description,
         };
       }),
-      events: getWebTypeEvents(component),
-      js: getJsProperties(component, options.typesSrc),
+      js: getJsProperties(component),
     };
   });
 }
 
-function getJsProperties(
-  component: Component,
-  typesSrc?: string
-): JsProperties {
+function getJsProperties(component: Component): JsProperties {
   return {
-    properties: getWebTypeProperties(component, typesSrc),
+    properties: getWebTypeProperties(component),
```

Comment:
Why was the `typeSrc` removed? This is used to provide custom types.

https://www.npmjs.com/package/custom-element-jet-brains-integration#custom-types

Reply:
I had assumed that was just an attribute thing since it wasn't used on `component.member`. After reading about the feature some more I can tell it should still be there. I can add back the equivalent functionality for getting the expanded type field from `component.member`.

Side note: I am curious, what is the motivation for having the expanded types field? Particularly for attributes that can only ever be of type `string`. I didn't see anything mentioned in the docs about what problem it was created to solve.

Reply:
This is often used to provide options for a union type. For example, if you had a button with an "appearance" property/attribute that accepts "default", "primary", "danger", or "success" as values, this and had them in a type called "ButtonAppearances", the Analyzer would store the type name and not the options. A lot of teams parse the type and store the options in the CEM as well to use in tooling and documentation. That's one of the reasons I created this tool.

https://www.npmjs.com/package/cem-plugin-expanded-types